keyboards/nebula68b/info.json

```
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "NEBULA68B",
+    "url": "",
+    "maintainer": "yiancar",
+    "layouts": {
+        "LAYOUT_68_ansi": {
+            "layout": [
+                {"x":0, "y":0},
+                {"x":1, "y":0},
+                {"x":2, "y":0},
+                {"x":3, "y":0},
+                {"x":4, "y":0},
+                {"x":5, "y":0},
+                {"x":6, "y":0},
+                {"x":7, "y":0},
+                {"x":8, "y":0},
+                {"x":9, "y":0},
+                {"x":10, "y":0},
+                {"x":11, "y":0},
+                {"x":12, "y":0},
+                {"x":13, "y":0},
+                {"x":13, "y":0},
```

Your keys are overlapping here.

```suggestion
                {"x":13, "y":0},
                {"x":14, "y":0},
```